server/aap/macros/preserve_format.py

```diff
@@ -33,7 +33,7 @@ def sanitize_tags(item):
         br.tail = '\n' + br.tail if br.tail else '\n'
     etree.strip_elements(parsed, 'br', with_tail=False)
 
-    for tag in parsed.xpath('/html/div/child::*'):
+    for tag in parsed.xpath('/div/child::*'):
```

Comment:
Where does this change, and all other `/html/` changes, come from? I cannot see it in any other branch.

Reply:
It came from this change to core https://github.com/superdesk/superdesk-core/commit/c2617aa5267399d871d5b0646036c13d03925739